HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -247,6 +247,20 @@ def customizeHLTfor42497(process):
 
     return process
 
+def customizeHLTfor42943(process):
+
+    for prod in producers_by_type(process, 'ClusterCheckerEDProducer'):
+        if hasattr(prod, "MaxNumberOfCosmicClusters"):
+            prod.MaxNumberOfStripClusters = getattr(prod,"MaxNumberOfCosmicClusters")
+            prod.__delattr__("MaxNumberOfCosmicClusters")
+
+    for prod in producers_by_type(process, 'SeedGeneratorFromRegionHitsEDProducer'):
+        if hasattr(prod, "ClusterCheckPSet"):
+            clustCheckPSet = getattr(prod,"ClusterCheckPSet")
+            prod.ClusterCheckPSet.MaxNumberOfStripClusters = getattr(clustCheckPSet,"MaxNumberOfCosmicClusters")
+            clustCheckPSet.__delattr__("MaxNumberOfCosmicClusters")
```

Comment:
Shouldn't this be "MaxNumberOfStripClusters" ?

Reply:
No.